autokeras/hypermodel/hyper_block.py

```diff
@@ -321,38 +366,70 @@ def func(x):
 
 class Flatten(HyperBlock):
 
+    def build(self, hp, inputs=None):
+        input_node = layer_utils.format_inputs(inputs, self.name, num=1)[0]
```

Comment:
Is there any difference between this and `tf.nest.flatten()`? The above is not very readable on its own (i.e. a reader of the code wouldn't immediately know what this does).